core/cmd/app.go

```diff
@@ -259,6 +259,20 @@ func NewApp(client *Client) *cli.App {
 					Usage:  "Run the chainlink node",
 					Action: client.RunNode,
 				},
+				cli.Command{
+					Name:   "p2p",
+					Usage:  "Local commands for administering the node's p2p layer",
+					Hidden: !client.Config.Dev(),
+					Subcommands: cli.Commands{
+						{
+							Name: "create",
+							Usage: format(`Create a p2p key, encrypted with password from the
+               password file, and store it in the database.`),
+							Flags:  flags("password, p"),
```

Comment:
The name of the flag `password` does not suggest it's a file path. To me it suggests it's the password in clear! Can you rename to `password-file` and update the Usage message accordingly. 
Also, `-p` seems to be a required field. Can you make that specific in the Usage message!

Reply:
I did it this way so it was the same as VRF generation